course_inventory/inventory.py

```diff
@@ -39,6 +39,8 @@
 INVENTORY_DB = ENV['INVENTORY_DB']
 APPEND_TABLE_NAMES = ENV.get('APPEND_TABLE_NAMES', ['job_run', 'data_source_status'])
 
+CANVAS_DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
```

Comment:
Isn't this format (ISO 8601 with `Z` shorthand for `+00:00`) already a constant in `datetime` or some similar module?

Reply:
It does seem to basically be the ISO Format. I didn't see this defined as a constant in `datetime`. Did you see it somewhere? This was hardcoded elsewhere in this file, I just extracted it to be a global variable since I needed it to.

Reply:
I've worked with ISO 8601 many times in Python and PHP, but I can't remember which language had this format assigned to a constant or a shortcut format for it.  I'll check around.  I remember that there was something that used a single character format, like `%c`, as a shortcut for `%Y-%m-%dT%H:%M:%SZ` or `%Y-%m-%dT%H:%M:%S%z`.

Reply:
Not gonna worry about this for now.